lib/csp.js

```diff
@@ -1,5 +1,5 @@
 var config = require('./config')
-var uuid = require('uuid')
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
This seems to be broken as this no module…
![image](https://user-images.githubusercontent.com/3978662/107996471-c564b100-6fe0-11eb-9fe1-263138b6c0f4.png)

Reply:
fixed, along with all the merge conflicts